client/html/templates/email/common/text-summary-detail-body-default.php

```diff
@@ -73,6 +73,7 @@
 <?php	$price = $product->getPrice(); ?>
 
 <?php echo strip_tags( $product->getName() ); ?>
+<?php echo strip_tags( $product->getProductCode() ); ?>
 
 <?php	foreach( array_merge( $product->getAttributes( 'config' ), $product->getAttributes( 'custom' ) ) as $attribute ) : ?>
 - <?php 	echo strip_tags( $this->translate( 'client/code', $attribute->getCode() ) ); ?>: <?php echo strip_tags( ( $attribute->getName() != '' ? $attribute->getName() : $attribute->getValue() ) ); ?>
```

Comment:
Sorry, I'll also keep this line (i.e. not remove it as it is shown in the diff!). It was removed locally to save vertical space. In general that might not be desirable.